lib/ui/pages/login.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:pispapp/controllers/ephemeral/login_controller.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/shadow_box.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+import 'package:pispapp/ui/widgets/bottom_button.dart';
+import 'package:pispapp/ui/widgets/phone_number.dart';
+
+class Login extends StatelessWidget {
+  Widget _loginWithGoogleTile() {
+    return ListTile(
+      contentPadding: const EdgeInsets.symmetric(),
+      title: const TitleText(
```

Comment:
Btw, I think it would be nicer if you create the typical "Login with Google" button like the one displayed [here](https://blog.codemagic.io/firebase-authentication-google-sign-in-using-flutter/), instead of just a normal text box.

Reply:
I wanted to make it consistent throughout the app. There is state involved too. On a successful login, we show a 'tick' sign.

Reply:
ohh I see, okay then